test-5.1.x.cfg

```diff
@@ -14,4 +14,6 @@ package-extras = [test]
 
 test-eggs =
 
-[versions]
```

Comment:
This file can probably be removed if there is only Plone 5.2 compatibility from now on.